specification/devopsinfrastructure/Microsoft.DevOpsInfrastructure.Management/client.tsp

```diff
@@ -7,3 +7,119 @@ using Azure.ClientGenerator.Core;
   "DevOpsInfrastructureMgmt",
   "python"
 );
+
+namespace Microsoft.DevOpsInfrastructure {
```

Comment:
instead of putting them in the same namespace, we can just import the types by `using Microsoft.DevOpsInfrastructure`

Reply:
fixed